apps/platform/src/app/projects/projects.service.ts

```diff
@@ -68,12 +70,17 @@ export class ProjectsService {
     return response;
   }
 
-  public getProjectSimulation(id: string): Observable<any> {
+  public getProjectSimulation(id: string): Observable<any> { // SimulationRun
     const url = this.endpoints.getSimulationRunEndpoint(id);
-    const response = this.http.get<string>(url).pipe();
+    const response = this.http.get<any>(url).pipe(shareReplay(1)); // SimulationRun
```

Comment:
The typing should be imported from the datamodel/api  lib

Reply:
This is held over from your version where some things were typed as `any`.